getting-started.adoc

```diff
@@ -1,3 +1,251 @@
 = Getting Started with Bisq
+:toc: left
+:sectanchors:
+:imagesdir: images
+:!figure-caption:
+:btc-deposit: 0.01
+:btc-deposit-plus-fees: 0.011
 
-COMING SOON. Subscribe to https://github.com/bisq-network/bisq-docs/issues/37[bisq-network/bisq-docs#37] for updates.
+Bisq is decentralized, private, and secure—but it's surprisingly approachable—and this guide will help you get up and running quickly.
+
+In this guide, we're going to assume you want to buy bitcoin in exchange for a national currency, since that's how most people get started with Bisq.
+
+Specifically, we'll show you how to buy bitcoin with US dollars, using a popular payment method called _Zelle_. Keep in mind, though, that Bisq offers support for trading all national currencies and dozens of different cryptocurrencies and tokens—the only constraint is that one side of each Bisq trade must always be bitcoin.
+
+== Download & install
+
+To run Bisq, download and install it from https://bisq.network/downloads/[the Bisq downloads page^]. _Optional:_ before installing, verify the signatures of the downloaded file (https://www.torproject.org/docs/verifying-signatures.html.en[instructions here^]) to be sure the file you downloaded is genuine.
+
+[NOTE]
+.Why do I need to download software? Why can't I just trade on your website?
+====
+Unlike most exchanges, Bisq doesn't run a centralized server for making and taking offers, so trading on a website isn't possible.
+
+Instead, Bisq manages offers to trade using a peer-to-peer network—a global network of people who are also running Bisq on their own computers.
+
+This is good news for users, because centralized services are easy to monitor, block and shut down, while peer-to-peer networks like BitTorrent, Bitcoin and Bisq are difficult to surveil, censor and hack.
```

Comment:
Initially this seemed to make the admonition a bit longer than I'd prefer, but considering it makes perhaps the most important point on the page, I think it's worth it. Great addition.